.prettierignore

```diff
@@ -10,3 +10,6 @@ target/
 pnpm-lock.yaml
 crates/next-core/js/src/compiled
 crates/turbopack/bench.json
+
+# This file has intentional trailing commas
+crates/turbopack/tests/node-file-trace/integration/ts-package/tsconfig.json
```

Comment:
Should we exclude any `**/tests/**` directory from prettier?

Reply:
I don't think so